pallets/committee/src/lib.rs

```diff
@@ -516,11 +517,8 @@ pub mod pallet {
 		pub fn set_voting_period(origin: OriginFor<T>, voting_period: T::BlockNumber) -> DispatchResult {
 			T::ApprovedByCommitteeOrigin::ensure_origin(origin)?;
 
-			let days = T::Days::get();
-			let weeks = days.saturating_mul(7u32.into());
-
 			ensure!(
-				weeks < voting_period && voting_period < weeks.saturating_mul(4u32.into()),
+				T::VotingPeriodRange::min() < voting_period && voting_period < T::VotingPeriodRange::max(),
```

Comment:
this should be `[min, max]` not `]min, max[` so that min and max are still in bounds